flow/result.go

```diff
@@ -80,16 +80,17 @@ func waitForSeal(ctx context.Context, c *client.Client, id flow.Identifier) (*fl
 	return result, nil
 }
 
-func printTxResult(tx *flow.Transaction, res *flow.TransactionResult) {
+func printTxResult(tx *flow.Transaction, res *flow.TransactionResult, showCode bool) {
 	fmt.Println()
 	fmt.Println("Status: " + res.Status.String())
 	if res.Error != nil {
 		fmt.Println("Execution Error: " + res.Error.Error())
-		return
 	}
 
-	fmt.Println("Code: ")
-	fmt.Println(string(tx.Script))
+	if showCode {
+		fmt.Println("Code: ")
+		fmt.Println(string(tx.Script))
+	}
 
 	fmt.Println("Events:")
```

Comment:
Events will now be printed even if there is an error (they will probably empty though) is that ok?

Reply:
Yeah, this was on purpose. This helped me with some debugging cases.